packages/sitecore-jss-nextjs/src/editing/editing-render-middleware.test.ts

```diff
@@ -236,8 +236,44 @@ describe('EditingRenderMiddleware', () => {
       expect(res.redirect).to.have.been.calledWith('/custom/path/styleguide');
     });
 
+    it('should handle request when sc_variant and sc_version are not passed', async () => {
```

Comment:
This test case might be extra, see my comment below regarding existing test case